src/RoslynPad.Roslyn/SymbolDisplayPartExtensions.cs

```diff
@@ -12,6 +13,9 @@ public static class SymbolDisplayPartExtensions
     {
         private const string LeftToRightMarkerPrefix = "\u200e";
 
+        public static Action<SymbolDisplayPartKind, Run> StyleRunFromSymbolDisplayPartKind { get; set; }
```

Comment:
`SymbolDisplayPartKind` is obsolete, I will be removing the overloads that use it soon.